### DISCUSSION TITLE
Hello Everyone

### DISCUSSION THREAD

**Discussion by dropy2008** [iptv-org/community#954] [Q&A]
hello guys, i need help from you if you can tell me how to make this correctly. So i run this SITE=tvpassport.com npm run grab, al good i running perfect , but in the guides folder i have like 4 different folder whit league indicative, how i can make this for exemple to be guide/usa/ tvpassport.com ?

**Comment by freearhey** [comment]
If I understood the question correctly you want to change the folder where the guide is saved to. If so, this can be done via the `--output` attribute.

https://github.com/iptv-org/epg?tab=readme-ov-file#usage
